src/pages/Booking/MusicalBooking/BookingProgress/BookingProgress.js

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { useBookingStore } from '../../store/store';
+import { ChevronRight } from 'react-feather';
+import { format } from 'date-fns';
+
+function BookingProgress({ seatSelectionStage, setSeatSelectionStage }) {
+  const bookingStatus = useBookingStore(state => state.bookingState);
+
+  const {
+    selectedMusical,
+    selectedTheatre,
+    selectedDate,
+    selectedTime,
+    selectedSeats,
+  } = bookingStatus;
+
+  const firstStepsComplete =
+    selectedMusical && selectedTheatre && selectedDate && selectedTime;
+
+  const allItemsSelected = Object.values(bookingStatus).every(v => v !== null);
+
+  return (
+    <BookingProgressStatusBar>
+      <div>
+        <BookingStepsContainer>
+          <BookingStep>
+            {selectedMusical?.id ? (
+              <MusicalInfo>
+                <AgeLimit ageLimit={selectedMusical.ageLimit}>
+                  {selectedMusical.ageLimit === 18 ||
+                  selectedMusical.ageLimit === 15 ||
+                  selectedMusical.ageLimit === 12
+                    ? selectedMusical.ageLimit
+                    : 'All'}
+                </AgeLimit>
```

Comment:
```suggestion
const ageLimit ={
18:18,
15:15,
12:12
}
  return (
    <BookingProgressStatusBar>
      <div>
        <BookingStepsContainer>
          <BookingStep>
            {selectedMusical?.id ? (
              <MusicalInfo>
                <AgeLimit ageLimit={selectedMusical.ageLimit}>
                  {ageLimit[selectedMusical.ageLimit]||'All'}
                </AgeLimit>
```
or 연산자가 중첩되서 조금 복잡해보일 수도 있는데, 여기도 어쨋든 비교하는식 이라 생각되서
객체로 해결해주면 어떨까 싶습니다.